server/src/main/java/org/apache/iotdb/db/metadata/MTree.java

```diff
@@ -657,7 +713,8 @@ private int getCountInGivenLevel(MNode node, int targetLevel) {
   /**
    * Iterate through MTree to fetch metadata info of all leaf nodes under the given seriesPath
    *
-   * @param timeseriesSchemaList List<timeseriesSchema>
+   * @param timeseriesSchemaList Collection<timeseriesSchema> result: [name, alias, storage group,
+   *                          dataType, encoding, compression, offset, lastTimeStamp]
    */
   private void findPath(MNode node, String[] nodes, int idx, String parent,
```

Comment:
it's better to return List<MeasurementMNode> instead of List<String>, Zesong may be working on this.

Reply:
It actually returns List, I forgot to change the comments back.